gfx/models/portraits/attachments/male_beards/western/male_beard_western_09_wc/male_beard_western_09_wc.asset

```diff
@@ -0,0 +1,25 @@
+pdxmesh = {
+	name = "male_beard_western_09_wc_mesh"
+	file = "male_beard_western_09_wc.mesh"
+
+	blend_shape = { id = "male_beard_western_09_bs_fat"		type = "male_beard_western_09_bs_fat_wc.mesh" }
+	blend_shape = { id = "male_beard_western_09_bs_old"		type = "male_beard_western_09_bs_old_wc.mesh" }
+
+	meshsettings = {
+		name = "|male_beard_western_09|male_beard_western_09|male_beard_western"
+		index = 0
+		texture_diffuse = "male_beard_western_09_diffuse_wc.dds"
+		texture_normal = "male_beard_western_09_normal_wc.dds"
+		texture_specular = "male_beard_western_09_properties_wc.dds"
+		shader = "portrait_hair"
+		shader_file = "gfx/FX/jomini/portrait.shader"
+	}
+}
+
+entity = {
+	name = "male_beard_western_09_wc_entity"
+	pdxmesh = "male_beard_western_09_wc_mesh"
+
+	attribute = { name = "bs_fat_1"					blend_shape = "male_beard_western_09_bs_fat" }
+	attribute = { name = "bs_old"					blend_shape = "male_beard_western_09_bs_old" }
+}
```

Comment:
What is changed here compared to the original one?

Reply:
> 
> 
> What is changed here compared to the original one?

I changed the name from `male_beard_western_09` to `male_beard_western_09_wc` for the necessary files. Although the only one that really needed to be changed is `male_beard_western_09_normal_wc.dds`, so if you want to change the others back, feel free to do so.

Reply:
Removed non needed files in 6a9a11084a216ca412cd293bfe3891259210b85c.